### PR TITLE
PS-7238 - Backport Data Masking plugin to 5.7

### PR DESCRIPTION
**Pull request by george-lorch**

### CODE REVIEW

**Comment by george-lorch** [comment]
https://ps57.cd.percona.com/job/percona-server-5.7-param/331/

File: mysql-test/mysql-test-run.pl [hunk 1]
```diff
@@ -7453,7 +7453,6 @@ ($)
   include-ndb[cluster]  Enable all tests that need cluster
   do-test=PREFIX or REGEX
                         Run test cases which name are prefixed with PREFIX
-                        or fulfills REGEX
```
Comment (dutow) [inline mysql-test/mysql-test-run.pl]: This is an unrelated change?

**Review by dutow — APPROVED** [review]

File: mysql-test/mysql-test-run.pl (hunk 1 above)
Comment (george-lorch) [reply mysql-test/mysql-test-run.pl]: oops, yes, unrelated but seems that it was in the 8.0 commit too /shrug  I'll pull it out
